models/user.js

```diff
@@ -0,0 +1,34 @@
+module.exports = (sequelize, { STRING }) => {
+  const User = sequelize.define(
+    'User',
```

Comment:
Is this a "User" or a "Channel"? I understand these have a 1-1 correspondence right now, but these are likely not the same thing as spee.ch continues to grow/evolve.

I think it is likely that this should be two tables, a User table and a Channel table, with Channel having a foreign key to User.

In addition to being more semantic, this would facilitate growth into a design where one user can access multiple channels from a single session/login without a tricky later refactor.

Reply:
per our conversation today, I am updating the db to include both User and Channel tables